src/gr11review/part1/Review1.java

```diff
@@ -1 +1,66 @@
 package gr11review.part1;
+
+import java.io.*;
+
```

Comment:
All classes should have a javadoc comment before the class definition to describe the class.  See https://docs.google.com/document/d/1OSzfpTEEmgVDhYGYWLsk9w2hsYxE92c8ba7amwvj6Kc/edit#heading=h.c18hp69vscvc